src/styles/reset.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  font-weight: 400px;
```

Comment:
use * selector for box-sizing: border:box only. For other rules, please indicate which properties you want to remove from  which exact tags